src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMessages.scala

```diff
@@ -60,4 +60,35 @@ object JmapMessages {
       .check(jsonPath("$.error").notExists)
       .check(jsonPath("$[0][1].messageIds[*]").findAll.saveAs("messageIds"))
 
+  def getRandomMessage() =
+    JmapAuthentication.authenticatedQuery("getMessages", "/jmap")
+      .body(StringBody(
+        """[[
+          "getMessages",
+          {
+            "ids": ["${messageIds.random()}"],
```

Comment:
It is not scala code. It is not a s"" string.

Here you see Gatling DSL on session access. 

By the way, there is no other way to do it.